README.md

```diff
@@ -1,7 +1,451 @@
-## Documentation
+# cOS
+
+**cOS** is an Immutable Adaptable Linux Distribution designed to reduce the maintenance surface exposed to the user. It is cloud-init driven and also designed to be adaptive-first, allowing easily to build changes on top.
+
+## In a nutshell
+
+cOS is built from Docker containers, and completely hosted on Docker registries. The build process results in a single Docker image used to deliver regular upgrades in OTA approach.
+
+cOS supports different release channels, all the final images used are tagged and pushed regularly [to DockerHub](https://hub.docker.com/r/raccos/releases-amd64/) and can be pulled for inspection from the registry as well. 
+Those are exactly the same images used during upgrades.
+
+For example, if you want to see locally what's in cOS 0.4.16, you can:
+
+```bash
+$ docker run -ti raccos/releases-amd64:cos-system-0.4.16
+```
+
+## Design goals:
+
+- Immutable distribution
+- Cloud-init driven
+- Based on systemd
+- Built and upgraded from containers - It is a [single image OS](https://hub.docker.com/r/raccos/releases-amd64/)!
+- OTA updates
+- Easy to customize
+
+## Quick start
+
+Download the ISO from the latest [release]() and run it in your virtualization hypervisor of choice (baremetal, too). You can login with the user `root` and `cos`. That's a live ISO and no changes will be persisted.
+
+Run `cos-installer <device>` to start the installation process. Remove the ISO and reboot.
+
+_Note_: `cos-installer` supports other options as well. Run `cos-installer --help` to see a complete help.
+
+## Upgrades / reset:
```

Comment:
This is already outdated, isn't it?

Reply:
I have rephrased it in https://github.com/mudler/cOS/pull/49/commits/c02424ef761280d59a45cb793bd837bef09e3f86 to avoid disambiguity :+1: 

It still explains about the `A/B` mechanism, but wasn't clear that was about images